### PR TITLE
Ks containers

### PR DESCRIPTION
**Pull request by harrysorensennrel**
Add tensorflow instructions

### CODE REVIEW

File: Containers/tensorflow.md [hunk 1]
```diff
@@ -0,0 +1,49 @@
+## Tensorflow with GPU support singularity container
+This singularity container supplies Tensorflow 2.3.0 optimized for use with GPU nodes.  It also has opencv, numpy, pandas, seaborn, scikit-learn python libraries.
```
Comment (KevinSayers) [inline Containers/tensorflow.md:2]: singularity -> Singularity

File: Containers/tensorflow.md [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+```
+4. Create singularity image file.  Note the ./images directory must be created before running this command.
+```bash
+docker run -v /var/run/docker.sock:/var/run/docker.sock \
+-v $(PWD)/images:/output \
+--privileged -t --rm \
+quay.io/singularity/docker2singularity --name tensorflow_custom.sif \
+tensorflow-custom-tag-name
+```
+5. Transfer image file to eagle.  For this example I created a directory named /scratch/$(USER)/tensorflow on eagle
```
Comment (KevinSayers) [inline Containers/tensorflow.md:46]: eagle -> Eagle

**Review by KevinSayers — COMMENTED** [review]
Just minor capitalization and missing Docker install link

**Comment by KevinSayers** [comment]
@harrysorensennrel quick follow up on this one, just some small changes and ready to merge.

File: Containers/tensorflow.md [hunk 3]
```diff
@@ -0,0 +1,49 @@
...
+pandas
+numpy
+scikit-learn
+git+https://github.com/tensorflow/docs
+```
+3. Build new docker image
+```bash
+docker build -t tensorflow-custom-tag-name .
+```
+4. Create singularity image file.  Note the ./images directory must be created before running this command.
```
Comment (diskwarrior) [inline Containers/tensorflow.md:38]:
```suggestion
4. Create Singularity image file.  Note the ./images directory must be created before running this command.
```

**Pull request merged**